sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.getCOESummary.e2e-spec.ts

```diff
@@ -128,6 +128,30 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-getCOESummary", ()
       });
   });
 
+  it("Should return a BadRequest error when the page number has an invalid integer.", async () => {
```

Comment:
Should we create a similar test for every endpoint with pagination also? Are all pagination endpoint using that same dto?

Reply:
I see the point. I believe that we do not need to revalidate the same over and over again but we do need to enforce that all pagination uses the same DTO. In the past, we did an effort to enforce this for all paginations. For this PR I did not check every single one.

Reply:
I did not see it as a critical point to be tested.
The intention was to have one test case added to validate the scenario. I would like to have it deployed to TEST as soon as possible to request a new Wava scan.